CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Unreleased
 
+### Fixed
+
+- `exclude_extra_fields`: A typo that prevented exclude configurations from being applied correctly
```

Comment:
This file is formatted in the way where lines are wrapped at 90 characters. It's also best if you do not reformat the rest of the file - this can be done in a separate PR where we can add relevant configuration too.

Reply:
Line wrapping should be fixed and I removed the reformat.

Reply:
Hmm I don't see your changes. Maybe something wrong with the merge with main?